templates/CRM/common/SocialNetwork.tpl

```diff
@@ -11,15 +11,15 @@
 
 <div class="crm-section crm-socialnetwork alert alert-success status crm-ok" role="alert">
     <h2>{ts}Help spread the word{/ts}</h2>
-    <p>{ts}Please help us and let your friends, colleagues and followers know about our page{/ts}{if $title}: <strong><a href="{$pageURL}">{$title}</a></strong>{else}.{/if}</p>
+    <p>{ts}Please help us and let your friends, colleagues and followers know about{/ts}{if $title}: <strong><a href="{$pageURL}">{$title}</a></strong>{else} {ts}our page{/ts}.{/if}</p>
```

Comment:
@kurund might be better to use repeat the string to insert variable in the string rather than cutting the string for better translation. Something like this might suffice: 
```

<p>
{if $title}
  {ts 1=$pageURL}Please help us and let your friends, colleagues and followers know about: <strong><a href="%1">{$title}</a></strong>{/ts}
{else}
  {ts}Please help us and let your friends, colleagues and followers know about our page.{/ts}
{/if}
</p>
```

Reply:
Thanks, I have made the corrections.